File: exs/mundo_1/golang/014.go
/*
Desafio 014

Problema: Escreva um programa que converta uma temperatura
          digitando em graus Celsius e converta para graus Fahrenheit.

Resolução do problema:
*/

package main

import(
	"fmt"
)

var temp float64

func main(){
	fmt.Print("Digite uma temperatura em Celsius: ")
	fmt.Scanln(&temp)

	f := 1.8 * temp + 32

	fmt.Println(temp, "graus Celsius em Fahrenheit fica", f)
}

File: exs/mundo_1/rust/014.rs
/*
Desafio 014

Problema: Escreva um programa que converta uma temperatura
          digitando em graus Celsius e converta para graus Fahrenheit.
*/

fn main() {
    let mut input = String::new();

    println!("Digite a temperatura em Celsius: ");
    std::io::stdin().read_line(&mut input).unwrap();

    let celsius: f32 = input.trim().parse().expect("Erro ao ler valor");

    let fahrenheit = (celsius * 9.0 / 5.0) + 32.0;

    println!("{celsius:.1}°C é igual a {fahrenheit:.1}°F");
}
